ui/components/multichain/multichain-token-list/multichain-token-list.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import classnames from 'classnames';
+import {
+  DISPLAY,
+  FLEX_DIRECTION,
+  FONT_WEIGHT,
+  JustifyContent,
+  Size,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import {
+  AvatarNetwork,
+  AvatarToken,
+  AvatarWithBadge,
+  AVATAR_WITH_BADGE_POSTIONS,
+  Text,
+} from '../../component-library';
+import Box from '../../ui/box/box';
+import { getNativeCurrencyImage } from '../../../selectors';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export const MultichainTokenList = ({
```

Comment:
I'm confused if this is going to be a list of items or a single item? Will you be adding logic to map items? I would suggest splitting it out into 2 components?

Reply:
So, the architecture here is - On the assets tab, by default we have the primary token that's `MultichainTokenList` (ETH) here. Then if we have other Tokens, we are passing them on to `TokenList`, it's mapped in `tokenCell` and then we are displaying each Token in MultichainTokenList.